src/Tickets/Commerce/Gateways/Contracts/Traits/Paid_Gateway.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Tickets Commerce: Paid Gateway Trait.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce\Gateways\Contracts\Traits
+ */
+
+namespace TEC\Tickets\Commerce\Gateways\Contracts\Traits;
+
+use TEC\Tickets\Commerce\Cart;
+
+trait Paid_Gateway {
+	/**
+	 * @inheritDoc
+	 */
+	public static function should_show() {
+		if ( is_admin() ) {
```

Comment:
```suggestion
		if ( is_admin() ) {
```
Is this worth adding a filter to control visibility?